swirl_lm/example/fire/fire_main.py

```diff
@@ -95,22 +95,57 @@
 - `ignition_scale`: The smoothness factor of the tanh function.
 """
 
-from absl import app
+from absl import app, flags
```

Comment:
Can you split app and flags into two separate lines? Our style guide requires imports to be on separate lines except for a few special cases (typing, etc.). I think you have to fix this in at least one more place.

Reply:
I have addressed this in [fccfdc7](https://github.com/google-research/swirl-lm/pull/3/commits/fccfdc7393e33e7a5b8f9e792864591aa856f7cf)